Codigo/CharacterPersistence.bas

```diff
@@ -381,7 +381,7 @@ Public Function LoadCharacterFromDB(ByVal userIndex As Integer) As Boolean
             If Not LoadCharacterInventory(UserIndex) Then Exit Function
             If Not LoadCharacterBank(UserIndex) Then Exit Function
             Call RegisterUserName(.id, .name)
-            Call Execute("update account set last_ip = ? where id = ?", .IP, .AccountID)
+            Call Execute("update account set last_ip = ? where id = ?", .ConnectionDetails.IP, .AccountID)
```

Comment:
This line can be removed now.

Reply:
I disagree, we need to fix this and be able to track user ip, we require this for the daily use case